index.html

```diff
@@ -0,0 +1,447 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/main.css" />
+    <link rel="icon" href="./favicon.ico" />
+    <title>Aziz Mejri - Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <h1 id="logo">Aziz Mejri</h1>
+      <button type="button" class="menu-bars" id="menu-bar">
```

Comment:
[Optional] According to MDN

![image](https://user-images.githubusercontent.com/74744787/157483354-a415c3fe-6e58-44e9-8b4d-ce682f651006.png)

it would be better if you just put h1 -h6 elements inside the header tag you can check [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).